deployments/tekton.go

```diff
@@ -484,6 +484,7 @@ func (k Tekton) createClusterRegistryCredsSecret(ctx context.Context, c *kuberne
 // in order to avoid pulling it the first time we an application is staged.
 // TODO: This doesn't work in a multi-node cluster because it will only pull
 // the image on one node. Maybe we could use a dummy daemonset for that.
+// This does not help with custom builder images.
```

Comment:
On a follow-up story, we should make it possible to change the default builder image permanently. This setting should either be persisted in the local epinio config file (thus it could be different for each epinio user), or globally (somewhere in the cluster).

In any case, I think we should completely remove this warm-up now, because:

- It only works on 1-node clusters (it's a pod not a daemonset).
- It only helps when using the default builder image.
- It's only needed on the first push.
- If needed, it can be handled outside `epinio install`. E.g. in our tests we could run a make target to warm up the cluster with this image. The users could do something similar on their clusters if they find the first push very important to be fast.

Reply:
Removed the job. Done. No need of make target since tests are passing.